### PR TITLE
Add call for experimental-CNI maintainers

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: docs/releases/1.26-NOTES.md [hunk 1]
```diff
@@ -65,3 +65,8 @@ CNIs, use the "cni" networking option instead.
 * Support for AWS Classic Load Balancer for API is deprecated and should not be used for newly created clusters.
 
 * All legacy addons are deprecated in favor of managed addons, including the [metrics server addon](https://github.com/kubernetes/kops/tree/master/addons/metrics-server) and the [autoscaler addon](https://github.com/kubernetes/kops/tree/master/addons/cluster-autoscaler).
+
+# Help Wanted
+
+* kOps needs maintainers for Canal, Flannel, and Kube-Router, to keep versions up to date and move the integration from experimental to stable.
+If no maintainers volunteer by kOps 1.27, support may be phased out.
```
Comment (hakman) [inline docs/releases/1.26-NOTES.md:72]: We need to remove support at the beginning of alphas and 1.27 is a little to early, in my opinion.
```suggestion
If no maintainers volunteer, support will be removed in kOps 1.28.
```

Comment (hakman) [reply docs/releases/1.26-NOTES.md:72]: I don't mind soft-removing them in 1.27, as we did with Weave, but would prefer to fully remove them in a future version, not wait until we remove support for k8s 1.27. WDYT?

Comment (johngmyers) [reply docs/releases/1.26-NOTES.md:72]: I would soft-remove CNIs, as with Weave. None of them are as out of date as Romana and Lyft were.

Comment (hakman) [reply docs/releases/1.26-NOTES.md:72]: OK

Comment (hakman) [inline docs/releases/1.26-NOTES.md:72]:
```suggestion
If no volunteers step up by the time kOps 1.27 is released, support will be phased out.
```

**Review by hakman — APPROVED** [review]

**Pull request merged**